### ISSUE TITLE
Borrow Direct Widget producing double-escaped URLs

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/requests#112]
The pass through links are coming in as:
```
https://borrow-direct.relaisd2d.com/search.html?query=ti%253D%2522trump%2522+and+au%253D%2522barrett%2522
```
Should be:
```
https://borrow-direct.relaisd2d.com/search.html?query=ti%3D%22trump%22+and+au%3D%22barrett%22
```
Sample Request: https://pulsearch-recap.princeton.edu/requests/10132742. 

Oddly enough when I check the log in umlaut (which is running the borrow_direct gem code) I can see that the link in the is done as:

```
Completed 200 OK in 168ms (Views: 0.5ms | ActiveRecord: 13.4ms)
Started GET "/link_router/1747" for 140.180.243.64 at 2017-03-22 17:48:51 -0400
Processing by LinkRouterController#index as HTML
  Parameters: {"id"=>"1747"}
Redirected to http://pulsearch-dev.princeton.edu/borrow-direct?query=ti%3D%22trump%22+and+au%3D%22barrett%22
Completed 302 Found in 16ms (ActiveRecord: 9.7ms)
```

@tampakis seems to be a problem on the OL side as well with queries that contain quotes. This could be an issue buried in the borrow_direct gem.

**Comment by kevinreiss** [comment]
The BD interface seems to like this form:

```https://borrow-direct.relaisd2d.com/search.html?query=ti%3D%22trump%22%20and%20au%3D%22barrett%22```

**Comment by kevinreiss** [comment]
Will be resolved when #121 is merged.
